Rubberduck.Core/UI/ToDoItems/ToDoExplorerControl.xaml

```diff
@@ -488,7 +490,7 @@
                 <Menu>
                     <MenuItem Header="{Resx ResxName=Rubberduck.Resources.RubberduckUI, Key=GroupingGrid_GroupingStyle}">
                         <MenuItem.Icon>
-                            <Image Height="16" Source="pack://application:,,,/Rubberduck.Resources;component/Icons/Custom/PNG/GroupBy.png" />
+                            <Image Height="16" Source="{StaticResource GroupByImage}" />
```

Comment:
all 16x16 fugue icons should be displayed at 16x16, otherwise they look off/weird. we don't have some `IconStyle` shared style somewhere that could be imported as a `ResourceDictionary` everywhere?